### ISSUE TITLE
Failing assert "orient" in to_json_args

### ISSUE THREAD

**Issue by evpinheiro** [ticdat/ticdat#202]
With the following PanDatFactory definition I'm getting the AssertionError `assert "orient" in to_json_args` while defining the schema. Do you have any clue what could be the problem?

```
> in module
    input_schema = PanDatFactory(
                   ^^^^^^^^^^^^^^
> in file __init__:
    self.json = pandatio.JsonPanFactory(self)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> in file __init__:
        assert "orient" in to_json_args
               ^^^^^^^^^^^^^^^^^^^^^^^^
```

<img width="568" alt="image" src="https://github.com/ticdat/ticdat/assets/17763599/542a8ebe-fa29-48b7-9c65-e516c64406c7">

**Comment by pjcpjc** [comment]
This is a `pandas` issue. You can back up to an older version of `pandas` or wait until my next internal (probably in a week or two). 

Looking over the docs, I believe `pandas` 2.2 broke this part of `ticdat`. I believe if you back your `pandas` up to  2.1 or earlier then you'll be fine. 

Thanks @evpinheiro  for bringing this to my attention.

**Comment by pjcpjc** [comment]
Closing because fixed on develop branch.  `ticdat` will see this fix in 0.2.24.5 or later
